src/reducers/versions.tsx

```diff
@@ -1523,17 +1542,35 @@ export const createReducer = ({
       }
       case getType(actions.setCurrentVersionId): {
         const { versionId } = action.payload;
-        let { expandedPaths } = state;
+        let { expandedPaths, versionInfo } = state;
+        const { selectedPath, visibleSelectedPath } = state;
         const version = getVersionInfo(state, versionId);
 
         if (version && expandedPaths) {
           expandedPaths = _getPrunedExpandedPaths(expandedPaths, version);
         }
 
+        // TODO: remove this conditional statement once no code depends
+        // on `selectedPath` and `visibleSelectedPath` in version
+        // See https://github.com/mozilla/addons-code-manager/issues/1258
+        /* istanbul ignore next */
```

Comment:
You should be able to add a test for this, right? I'd think that you could make an assertion to compare `version.selectedPath` to `selectedPath`. If so, it would be better to add test coverage for it even if we will delete the tests later after removing the functionality.